storagemarket/provider_loop.go

```diff
@@ -132,30 +130,30 @@ func (p *Provider) loop() {
 			go func() {
 				defer p.wg.Done()
 				p.doDeal(deal, dealReq.dh)
+				p.dealLogger.Infow(deal.DealUuid, "deal sent for execution")
```

Comment:
```suggestion
				p.dealLogger.Debugw(deal.DealUuid, "deal sent for execution")
```

Reply:
This will be just once per deal and helpful in tracing code flow, so no harm in keeping it.